pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = [ "setuptools", "setuptools_scm" ]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ts_guitool"
+description = """
+This package is the Python QT tool code."""
+license = { text = "GPL" }
+classifiers = [ "Programming Language :: Python :: 3" ]
+urls = {documentation = "https://jira.lsstcorp.org/secure/Dashboard.jspa", repository = "https://github.com/lsst-ts/ts_guitool" }
```

Comment:
```suggestion
urls = {documentation = "https://rubinobs.atlassian.net/secure/Dashboard.jspa", repository = "https://github.com/lsst-ts/ts_guitool" }
```

Reply:
Updated in the commit: [c2950b7](https://github.com/lsst-ts/ts_guitool/pull/1/commits/c2950b7e377381a298a81fd240ce595db475b8d9). Thanks!